sdk/cosmosdb/cosmos/src/CosmosClient.ts

```diff
@@ -81,6 +84,24 @@ export class CosmosClient {
       throw new Error("Invalid endpoint specified");
     }
 
+    if (optionsOrConnectionString.encryptionPolicy?.enableEncryption) {
+      if (!optionsOrConnectionString.encryptionPolicy.keyEncryptionKeyResolver) {
+        throw new Error(
+          "KeyEncryptionKeyResolver needs to be provided to enable client-side encryption.",
+        );
+      }
+      if (!optionsOrConnectionString.encryptionPolicy.encryptionKeyResolverName) {
+        throw new Error(
+          "EncryptionKeyResolverName needs to be provided to enable client-side encryption.",
+        );
+      }
+      this.encryptionManager = new EncryptionManager(
+        optionsOrConnectionString.encryptionPolicy.keyEncryptionKeyResolver,
+        optionsOrConnectionString.encryptionPolicy.encryptionKeyResolverName,
+        optionsOrConnectionString.encryptionPolicy.encryptionKeyTimeToLive,
```

Comment:
why there is no check for encryptionKeyTimeToLive? is it not required?

Reply:
Default value(2 hours) is picked if no value is provided.